include/chemfiles/periodic.hpp

```diff
@@ -14,140 +13,144 @@
 
 namespace chemfiles {
 
-/// Storing basic elemental data: mass, colvalent and Van der Waals radii
+//! Storing basic elemental data: mass, colvalent and Van der Waals radii, and default element colors for viewers
 struct ElementData {
-    /// Atomic number
+    //! Atomic number
     const int number;
-    /// Full name
+    //! Full name
     const char* name;
-    /// Mass in atomic units
+    //! Mass in atomic units
     const double mass;
-    /// Covalent radius in Angstrom
+    //! Covalent radius in Angstrom
     const double colvalent_radius;
-    /// Van der Waals radius in Angstrom
+    //! Van der Waals radius in Angstrom
     const double vdw_radius;
+    //! Default element colors for viewers
+    const float elementColor_r;
```

Comment:
Following the same idea of using a single vector for the three components of the color, this could be `const double[3] rgb_color`.